portfolio-project-matching-app/components/ProjectCard.js

```diff
@@ -17,32 +17,36 @@ const ProjectCard = ({ project }) => {
                 <p className='text-xl font-medium'>{project.name}</p>
             </div>
             <hr className='w-11/12 sm:w-9/12 border-b-2 border-gray-400'/>
-            <div className='flex flex-wrap'>
+            {/* Commented out, but leaving in just in case we decide to add this back */}
+            {/* <div className='flex flex-wrap'>
                 {project.technologies.map((technology) => {
                     return (
                         <div key={technology.id} className='py-1 pr-2'>
                             <Button text={technology.name}/>
                         </div>
                 )
                 })}
-            </div>
+            </div> */}
             <div className=''>
                 <p>{project.description}</p>
             </div>
```

Comment:
How would we feel about removing the description from the `ProjectCard`? Or only displaying a certain number of characters for a more brief version? The only reason I say this is there are some descriptions that are lengthy (Like the "Bet With Your Friends" Project)

Reply:
Hey Charlie - my recent pushes should reflect changes in the direction of this comment! Let me know what you think. The description above should give you an idea. Also, I posted some snippets in our Slack chat.